client/packages/system/src/Name/Details/Details.tsx

```diff
@@ -186,6 +226,29 @@ export const Details = ({ nameId, type = 'customer' }: DetailsProps) => {
             }
             labelWidthPercentage={19}
           />
+          {type === 'customer' && (
```

Comment:
Oh this was an extra request? Can this be read only please thanks

Reply:
Oh I might have misunderstood the requirement. This was my interpretation of the second point. Would that not be the case anymore? 

<img width="548" height="137" alt="Image" src="https://github.com/user-attachments/assets/b87feb7a-ae00-4ee1-a968-cbb1696f34f2" />